cmd/report.go

```diff
@@ -107,3 +321,16 @@ func writeNodeReport(records []reporting.NodeReportItem) {
 	}
 	table.Render()
 }
+
+func writeErrorBuilders(errBuilders ...strings.Builder) {
```

Comment:
I think instead of writing all the errors to the STDERR we should store them on disk and let the user know where to find the errors. If we don't want this to be the default behavior, then at least offer a parameter like `--error-file foo.out`

Reply:
Created https://github.com/chef/chef-analyze/issues/82